### PR TITLE
fixes #15164 - user can specify org/loc for proxy registration

### PR DESCRIPTION
**Pull request by stbenjam**
Needs https://github.com/theforeman/puppet-foreman/pull/448

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -257,6 +257,11 @@
   $registered_name = $::fqdn
   # Proxy URL to be registered
   $registered_proxy_url = undef
+
```
Comment (ekohl) [inline manifests/params.pp]: Nit: since this is the registration block, you can leave out this empty line.

**Comment by ekohl** [comment]
We shouldn't forget to increase the minimum puppet-foreman version.

**Comment by stbenjam** [comment]
@ekohl I think that's done, no? metadata.json shows puppet-foreman 5.2, which hasn't be released yet.  If my PR doesn't make it for 5.2.0, it will need updating though.

**Comment by stbenjam** [comment]
[test]

**Comment by mmoll** [comment]
[test] won't work with travis jobs, but it would fail without the corresponding puppet-foreman PR, correct?

**Comment by stbenjam** [comment]
Didn't realize [test] wouldn't trigger it.   Anyway, I realize it'll fail was just curious what was going on with the travis job, it never actually ran.

**Comment by mmoll** [comment]
Ah, I canceled it, as our account travis was/is a bit swamped by all the long test runs

**Comment by ekohl** [comment]
Given https://github.com/theforeman/puppet-foreman/pull/448 was closed I'm closing this as well.

**Pull request closed without merging**